doc/qbk/0.main.qbk

```diff
@@ -73,8 +73,8 @@
 [def __params_encoded__         [link url.ref.boost__urls__params_encoded `params_encoded`]]
 [def __params_encoded_view__    [link url.ref.boost__urls__params_encoded_view `encoded_params_view`]]
 [def __pchars__                 [link url.ref.boost__urls__pchars `pchars`]]
-[def __pct_encoded_view__       [link url.ref.boost__urls__pct_encoded_view `pct_encoded_view`]]
-[def __pct_encoded_rule__       [link url.ref.boost__urls__pct_encoded_rule `pct_encoded_rule`]]
+[def __decode_view__            [link url.ref.boost__urls__encoded_view `decode_view`]]
+[def __encoded_rule__           [link url.ref.boost__urls__encoded_rule `encoded_rule`]]
```

Comment:
I wonder if we should leave the rule alone, what do you think? It isn't going to be used much, so the verbosity isnt' so bad. And the name "pct-encoded" comes from the RFC (its the name of the grammar).